packages/frontend/pages/index.vue

```diff
@@ -18,57 +13,89 @@ GNU Affero General Public License for more details.
-        </h5>
- 
- 
+    <div class="container-fluid">
+
+        <div class="row" id="first-row">
+            <div class="col-12 col-md-7" id="first-row-col">
+                <div class="row"> <h1>Trust and transparancy when you need it most.</h1> </div>
+                <div class="row" style=" margin-bottom: 60px; margin-top:20px; display:inline-flex">
+                    <form style="margin-bottom: 20px; width:40%; width: 190px; padding-right: 0px;" @submit.prevent="trackingForm">
+                        <button-component class="button" id="trackButton" buttonText="View Record" type="submit" style="opacity:100;"
+                            padding="18px 22px"></button-component>
+                    </form>
+                    <div style="margin-bottom: 20px; width: 60%; width: 230px;" >
+                        <button-component class="button" buttonText="Create Record" backgroundColor="#CCECFD" onclick="window.location.href='/gdt'"
+                            borderColor="#CCECFD" color="#1E2019" padding="18px 22px" margin="0px 0px 0px 0px"></button-component>
+                    </div>
+                    <div id="homeTrackAssetDiv" style="visibility: hidden;">
+                        <TrackAsset inputWidth="75%"></TrackAsset>
+                    </div>
+                </div>
+                
+            </div>
+
+        </div>
+
+        <div class="row bg-frost" id="second-row" >
+            <div class="row rowtest">
+                <div class="col test" id="second-row-cols" v-for="item in second_row">
+                    <h3 class="text-iris" id="second-row-cols-h3">{{item.title}}</h3>
+                    <p class="text-eggplant" style="font-weight: 400;">{{ item.descr }}</p>         
+                </div>
+            </div>
+            <div class="col" style="text-align: center; margin-top: 50px;">
+                <button-component class="button" buttonText="About Us" color="#322253" onclick="window.location.href='about'"
+                    backgroundColor="#ffffff00"></button-component>
+            </div>
+
+        </div> 
+
+        <Learn_more id="learn-more"></Learn_more>
+
     </div>
- </template>
+
+</template>
+
+
 
 <script lang="ts">
+import Learn_more from '~/layouts/learn_more.vue';
+import ButtonComponent from '~/components/ButtonComponent.vue';
+let showTrack = false;
+
+const second_row = [
+    { title: "Simplicity & Accessibility", descr: "We believe that open-source projects should be simple to use and understand."},
+    { title: "Data Ownership", descr: "We do not have access to any user data, ensuring complete privacy and independent ownership."},
```

Comment:
Please can you use the full word "description"

Reply:
If you use the word 'description' instead of 'descr' then the text doesn't appear on the screen.